src/meta/app/src/schema/table.rs

```diff
@@ -616,3 +616,45 @@ pub struct CountTablesReply {
 pub struct TableIdToName {
     pub table_id: u64,
 }
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Eq, PartialEq, Default)]
+pub struct TableCopiedFileNameIdent {
+    pub table_id: u64,
+    pub file: String,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Eq, PartialEq, Default)]
+pub struct TableCopiedFileInfo {
+    pub etag: Option<String>,
+    pub content_length: u64,
+    pub last_modified: Option<DateTime<Utc>>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct GetTableCopiedFileReq {
+    pub table: TableNameIdent,
+    pub files: Vec<String>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct GetTableCopiedFileReply {
+    pub file_info: BTreeMap<String, TableCopiedFileInfo>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct UpsertTableCopiedFileReq {
+    pub table: TableNameIdent,
+    pub file_info: BTreeMap<String, TableCopiedFileInfo>,
+    pub expire_at: Option<u64>,
+}
```

Comment:
GetTableCopiedFileReq and UpsertTableCopiedFileReq will be convert to using table-id, too?